test.js

```diff
@@ -29,18 +29,18 @@ describe('<message>', () => {
     it('throws error when ":" token is missing', () => {
       expect(() => {
         parser('feat add support for scopes')
-      }).to.throw("unexpected token ' ' at position 1:5 valid tokens [(, !, :]")
+      }).to.throw("unexpected token ' ' at 1:5, valid tokens [(, !, :]")
```

Comment:
Position itself is a pretty distinct format, keeping this short and descriptive is important. But, feel free to roll this rewording back!

Reply:
I'm supportive of this rewording.